RELEASING.md

```diff
@@ -72,6 +74,62 @@ and deadlocks.
     (note that if this is not a patch release then the change log on main may already be up-to-date,
     in which case no pull request will be created).
 
+## Breaking Changes and Deprecations Automation
```

Comment:
I think I'd remove this and add the minimum needed for contributors in the CONTRIBUTING.md